assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -47,4 +47,9 @@ const content = (
   </Provider>
 );
 
-ReactDOM.render(content, document.getElementById('bundles-landing-page'));
+const element: ?Element = document.getElementById('bundles-landing-page');
+
+if (element) {
+  ReactDOM.render(content, element);
+}
```

Comment:
Worth using a helper function for this?

Reply:
I think a helper would be a good idea here to remove code duplication

Reply:
I didn't like the idea of creating a helper function for an if clause but since it is repeated in more than 5 files I will add it.

Reply:
If this check fails, we render a blank page but do not throw an error.  We should be logging this to sentry